plugin/completion.py

```diff
@@ -78,11 +78,14 @@ def update_popup(self, minihtml_content: str) -> None:
 
 class LspCompleteCommand(sublime_plugin.TextCommand):
 
-    def handle_additional_edits(self, item: Dict[str, Any]) -> None:
+    def epilogue(self, item: Dict[str, Any]) -> None:
```

Comment:
Nitpic 🙂 maybe rename to on_completion_inserted or something like that? 

Epilogue reminds me a lot on  the word you offend use: vague